src/client/public/config/openfin/launcher-dev.json

```diff
@@ -1,80 +1,85 @@
-    "defaultTop": 160,
-    "defaultLeft": 30,
-    "autoShow": true,
-    "saveWindowState": false,
-    "resizable": false,
-    "shadow": true,
-    "frame": false,
-    "backgroundColor": "#444C5F",
-    "alwaysOnTop": true,
-    "icon": "https://web-dev.adaptivecluster.com/static/media/icon.ico",
-    "applicationIcon": "https://web-dev.adaptivecluster.com/static/media/adaptive-mark-large.png",
-    "contextMenu": true,
-    "_comment": "Openfin Excel API preloaded below + added in appAssets (not included in standard OpenFin package)",
-    "preload": [
-      {
-        "url": "https://web-dev.adaptivecluster.com/plugin/service-loader.js"
-      },
-      {
-        "url": "https://web-dev.adaptivecluster.com/plugin/fin.desktop.Excel.js"
-      }
-    ],
-    "accelerator": {
-      "devtools": true,
-      "reload": true,
-      "reloadIgnoringCache": true,
-      "zoom": true
-    }
-  },
-  "runtime": {
-    "arguments": "--remote-debugging-port=9222",
-    "version": "13.76.44.21"
-  },
-  "shortcut": {
-    "company": "Adaptive Consulting",
-    "icon": "https://web-dev.adaptivecluster.com/static/media/icon.ico",
-    "name": "Reactive Ecosystem Launcher (DEV)"
-  },
-  "services": [
-    {
-      "name": "layouts",
-      "config": {
-        "features": {
-          "snap": true,
-          "dock": true,
-          "tab": false
+    "startup_app": {
+        "name": "Reactive Ecosystem Launcher (DEV)",
+        "url": "https://web-dev.adaptivecluster.com/launcher",
+        "uuid": "ReactiveLauncher",
+        "defaultWidth": 650,
+        "defaultHeight": 52,
+        "defaultTop": 160,
+        "defaultLeft": 30,
+        "autoShow": true,
+        "permissions": {
+            "System": {
+                "launchExternalProcess": true
+            }
+        },
+        "saveWindowState": false,
+        "resizable": false,
+        "shadow": true,
+        "frame": false,
+        "backgroundColor": "#444C5F",
+        "alwaysOnTop": true,
+        "icon": "https://web-dev.adaptivecluster.com/static/media/icon.ico",
+        "applicationIcon": "https://web-dev.adaptivecluster.com/static/media/adaptive-mark-large.png",
+        "contextMenu": true,
+        "_comment": "Openfin Excel API preloaded below + added in appAssets (not included in standard OpenFin package)",
+        "preload": [
+            {
+                "url": "https://web-dev.adaptivecluster.com/plugin/service-loader.js"
+            },
+            {
+                "url": "https://web-dev.adaptivecluster.com/plugin/fin.desktop.Excel.js"
+            }
+        ],
+        "accelerator": {
+            "devtools": true,
+            "reload": true,
+            "reloadIgnoringCache": true,
+            "zoom": true
         }
-      },
-      "manifestUrl": "https://cdn.openfin.co/services/openfin/layouts/1.1.0/app.json"
     },
-    {
-      "name": "fdc3",
-      "manifestUrl": "https://cdn.openfin.co/services/openfin/fdc3/0.2.0/app.json"
+    "runtime": {
```

Comment:
am I right that you only added permissions as we discussed on the call?

Reply:
yup

Reply:
thanks, the diff is confusing